server/jetstream.go

```diff
@@ -1522,24 +1522,24 @@ func (jsa *jsAccount) checkBytesLimits(addBytes int64, storage StorageType, repl
 		// Account limits defined.
 		if jsa.limits.MaxMemory > 0 {
 			if jsa.memReserved+totalBytes > jsa.limits.MaxMemory {
-				return ApiErrors[JSMemoryResourcesExceededErr]
+				return NewJSMemoryResourcesExceededError()
 			}
 		} else {
 			// Account is unlimited, check if this server can handle request.
 			if js.memReserved+addBytes > js.config.MaxMemory {
-				return ApiErrors[JSMemoryResourcesExceededErr]
+				return NewJSMemoryResourcesExceededError()
 			}
 		}
 	case FileStorage:
 		// Account limits defined.
 		if jsa.limits.MaxStore > 0 {
 			if jsa.storeReserved+totalBytes > jsa.limits.MaxStore {
-				return ApiErrors[JSStorageResourcesExceededErr]
+				return NewJSMemoryResourcesExceededError()
```

Comment:
Should it be NewJSStorageResourcesExceededError here? (and the one below)

This would solve this test failure:
```
=== RUN   TestJetStreamClusterMaxBytesForStream
    jetstream_cluster_test.go:964: Expected "insufficient storage resources" error, got "insufficient memory resources available"
--- FAIL: TestJetStreamClusterMaxBytesForStream (1.70s)
```

Reply:
Yes - I am sure there's a few of these, this is what I meant with carefully `carefully double check some stuff` :)